exampleSite/tailwind.config.js

```diff
@@ -0,0 +1,36 @@
+const colors = {
```

Comment:
So tailwind.config.js would be in addition to style changes within config.toml? That sounds a bit more advanced. Can we grab the colors from config.toml instead?
Any other way to reduce the need to have a tailwind.config.js?